dts/arm/nxp/nxp_rt10xx.dtsi

```diff
@@ -1151,3 +1151,7 @@
 	 */
 	status = "disabled";
 };
+
+nxp_cam_i2c: &lpi2c1 {};
```

Comment:
This probably should be in the board level overlay.

Reply:
yes, done